GeosCore/input_mod.F90

```diff
@@ -5600,6 +5600,19 @@ SUBROUTINE CHECK_TIME_STEPS( Input_Opt, State_Grid, RC)
     TS_DYN  = Input_Opt%TS_DYN
     TS_RAD  = Input_Opt%TS_RAD
 
+      ! If we're doing the reverse integration
+      ! multiply all the timesteps by -1 here
+      if (TS_DYN < 0) THEN
+         ! TS_DYN and TS_CHEM should always be set to something valid...?
+         TS_DYN = TS_DYN * -1
+         TS_CHEM = TS_CHEM * -1
+         if (LTURB .or. LCONV) TS_CONV = TS_CONV * -1
+         if (LEMIS .or. LDRYD) TS_EMIS = TS_EMIS * -1
+         ! I'm not sure which flag determins radiation on/off?
+         if (LCHEM) TS_RAD = TS_RAD * -1
+      endif
+         
+
```

Comment:
For GC-Classic to build and run I needed to change this section to this:
```
    ! If we're doing the reverse integration                                              
    ! multiply all the timesteps by -1 here                                               
    IF ( TS_DYN < 0 ) THEN
       TS_CHEM = TS_CHEM * -1
       TS_EMIS = TS_EMIS * -1
       TS_CONV = TS_CONV * -1
       TS_DYN  = TS_DYN  * -1
       TS_RAD  = TS_RAD  * -1
    ENDIF

```

Reply:
Otherwise this looks good to go. If you could just push the update I will go ahead and merge. Thanks for your work on this!